core/bin/sqlite_worker.rs

```diff
@@ -142,12 +142,18 @@ impl WorkerState {
             Err(_) => Err(anyhow!("CORE_API not set."))?,
         };
 
+        let core_api_key = match std::env::var("CORE_API_KEY") {
```

Comment:
We want this in a lazy static no? I thiiink env::var is a "but" blocking no?

Reply:
fair 👍  then will do for all env vars in this file